openwhisk/executor.go

```diff
@@ -44,13 +44,16 @@ type Executor struct {
 // NewExecutor creates a child subprocess using the provided command line,
 // writing the logs in the given file.
 // You can then start it getting a communication channel
-func NewExecutor(logout *os.File, logerr *os.File, command string, args ...string) (proc *Executor) {
+func NewExecutor(logout *os.File, logerr *os.File, command string, env map[string]string, args ...string) (proc *Executor) {
 	cmd := exec.Command(command, args...)
 	cmd.Stdout = logout
 	cmd.Stderr = logerr
 	cmd.Env = []string{
```

Comment:
unrelated to this PR... but is this __OW_API_HOST export necessary? it's already in the environment.

Reply:
if it is already in the Env map, then no...

Reply:
It’s set on the container when it’s started via docker.

Reply:
Hmm but in this case I have to propagate it to the subprocess I lanunch... so it was wrong to remove it...

Reply:
just to be clear, when the invoker starts the container, it uses `docker run -e __OW_API_HOST ...` so it's available in the container already.

Reply:
Ouch! So we have to re-add it...

Reply:
Hmm but if  in the json there is the { "apihost": ... } field then it is ok... it was reduntant